google-cloud-spanner/Rakefile

```diff
@@ -98,7 +100,7 @@ namespace :acceptance do
     $LOAD_PATH.unshift "lib"
     require "google/cloud/spanner"
     puts "Cleaning up Spanner instances and databases."
-    Google::Cloud::Spanner.new.instances.all do |instance|
+    Google::Cloud::Spanner.new(emulator_host: emulator_host).instances.all do |instance|
```

Comment:
Same here. If `SPANNER_EMULATOR_HOST` is set, we should not need to pass it as a parameter: 

https://github.com/googleapis/google-cloud-ruby/blob/5f61bf15a744f8d969e6bdae80f57ac8ea150d5b/google-cloud-spanner/test/google/cloud/spanner_test.rb#L325-L341

Reply:
done